tests/testAutopas/testingHelpers/RandomGenerator.h

```diff
@@ -44,6 +47,24 @@ class RandomGenerator {
   static void fillWithParticles(Container &container, const Particle &defaultParticle,
                                 unsigned long numParticles = 100ul, unsigned int seed = 42);
 
+  /**
+   * Fills the given container with randomly distributed particles between boxMin and boxMax.
+   * Adapted for md-flexible for initializing domain on specific Particle ID and Particle Type
+   * @tparam Container
+   * @tparam Particle Type of particle to be generated (got to inherite of MoleculeLJ)
+   * @param container
+   * @param typeId
+   * @param id
+   * @param defaultParticle inserted particle
+   * @param boxMin min. position
+   * @param boxMax max. position
+   * @param numParticles number of particles
+   */
+  template <class Container, class Particle>
+  static void fillWithParticles(Container &container, size_t typeId, size_t id, const Particle &defaultParticle,
+                                const std::array<double, 3> &boxMin, const std::array<double, 3> &boxMax,
+                                unsigned long numParticles = 100ul);
+
```

Comment:
we actually do not need this version, since you can also set the typeId in the default particle that is passed to the existing function. This also applies to the extra versions in the other generator classes.